test/test_deployment_utils.py

```diff
@@ -0,0 +1,395 @@
+import datetime
+import os
+
+import pytest
+import re
+import subprocess
+
+from contextlib import contextmanager
+from io import StringIO
+from unittest import mock
+
+from dcicutils.deployment_utils import Deployer
+from dcicutils.misc_utils import ignored
+from dcicutils.env_utils import is_cgap_env
+
+
+_MY_DIR = os.path.dirname(__file__)
+
+
+class TestDeployer(Deployer):
+    TEMPLATE_DIR = os.path.join(_MY_DIR, "ini_files")
+    PYPROJECT_FILE_NAME = os.path.join(os.path.dirname(_MY_DIR), "pyproject.toml")
+
+
+@contextmanager
+def override_environ(**overrides):
```

Comment:
Yeah, that seems worth doing. The module you're looking for is called `qa_utils`, by the way, not `test_utils`, so that it doesn't look like it's testing some file called `utils.py` (sigh).